### ISSUE TITLE
Remove wiki in favor of read the docs

### ISSUE THREAD

**Issue by csoriano89** [flatpak/flatpak#731]
"Official" documentation or information about Flatpak can be found currently in flatpak.org, flatpak wiki (https://github.com/flatpak/flatpak/wiki), flatpak GNOME wiki (https://wiki.gnome.org/Projects/SandboxedApps) and read the docs (http://docs.flatpak.org/en/latest/).

This is suboptimal, since it's even hard to keep the documentation relevant and up to date in a single place. Not only that, but the outdated info in the Flatpak wiki raised some concerns and misunderstandings recently, believing that Flatpak relies on SElinux (so not ootb crossplatform).

I think we can say that the Flatpak website (flatpak.org) is a nice way to get the first touch, so the info there, even if replicated, is good.
On the other hand, the "read the docs" documentation is nicely written and displayed and has all the information present on all the other websites mentioned above. Not only that, but the command line documentation is also in there.

So my proposal is to remove all the other pages, making sure no info is lost, and make "official" only the Flatpak website and the read the docs manual.

TODO would be:
A link in http://flatpak.org/index.html#features should be removed
Remove/update https://freedesktop.org/wiki/Software/Flatpak/
Remove https://github.com/flatpak/flatpak/wiki
Remove https://wiki.gnome.org/Projects/SandboxedApps

**Comment by matthiasclasen** [comment]
I don't understand what your problem with https://wiki.gnome.org/Projects/SandboxedApps is - it is already just a stub that refers to other places.

**Comment by csoriano89** [comment]
True, since it links to other places in the GNOME wiki maybe that one is fine.
Just trying to eliminate possible places where we have to maintain docs/links.

**Comment by matthiasclasen** [comment]
The same goes for https://freedesktop.org/wiki/Software/Flatpak/ - it is just a stub that points you in the right direction.

**Comment by matthiasclasen** [comment]
The one case that may be interesting to look at is https://github.com/flatpak/flatpak/wiki
I don't know how the content in there overlaps with readthedocs.

**Comment by matze** [comment]
Even if there's no overlap, it's a bit frustrating to have technical information scattered in those two places. I had to refer to the tips & tricks section multiple times because information there was not readily available and linked properly in the rtd docs.

**Comment by csoriano89** [comment]
Yes, one of my TODO is to put that info somewhere in "read the docs"
